plugins/woocommerce/src/Internal/Features/FeaturesController.php

```diff
@@ -393,7 +393,27 @@ private function get_feature_definitions() {
 					'is_legacy'          => true,
 					'is_experimental'    => false,
 				),
-				'block_email_editor'     => array(
+				'reactify-classic-payments-settings' => array(
+					'name'               => __( 'Payment Settings (beta)', 'woocommerce' ),
+					'description'        => __(
+						'Enable the new payments settings experience',
```

Comment:
Nitpick: I think `payment settings` instead of `payments settings` reads more naturally.

Reply:
Thanks for the nitpick @dmallory42 . Now if we consider that the tab label is "Payments" would you not call the page "Payments settings"? Same as for "Products settings"? Because it is about payments in general, not a specific gateway (probably there it makes more sense to use the singular).